TestHelpers.Tests/MockDirectoryGetAccessControlTests.cs

```diff
@@ -16,6 +16,11 @@ public class MockDirectoryGetAccessControlTests
         [TestCase("   ")]
         public void MockDirectory_GetAccessControl_ShouldThrowArgumentExceptionIfPathContainsOnlyWhitespaces(string path)
         {
+            if (MockUnixSupport.IsUnixPlatform()) 
```

Comment:
Instead of this block of code throughout the test project, what do we think about categorizing Windows tests?

I think this would declutter the test runner a bit more (no need for inconclusives) and would also clearly show that the feature we're testing is intended for Windows.